modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,88 @@
+
+== Introduction
+
+If you have a QNAP NAS (Network Attached Storage) system, you can install an ownCloud docker image from the QNAP App Center on your NAS. The free-of-charge community edition can be used with up to five users and an unlimited number of guest users.
+
+In case you want to use Enterprise features, you can purchase an Enterprise license from the QNAP store. This also allows you to purchase licenses for more users.
+
+The QNAP ownCloud is designed to work out of the box. We strongly recommend not to change the database (PostgreSQL) or the Web server (Apache). All configuration should happen in the Web interface. However command line configuration via `occ` commands is availabe.
+
+== Preparation
+
+If you haven't done so already, perform the following steps on you QNAP system:
+
+. Create a storage pool in addition to the system volume. You can combine hard disks and solid state disks in RAID groups for flexible storage. This pool can be resized later if necessary.
+
+. In the AppCenter, go to the QNAP Store and install Container Station version 2.0 or higher, which provides virtualization and Docker technogies. 
+
+. Enable `ssh` (secure shell) in the ControlPanel. In the navigation bar on the left side, select menu:Networks & File Services[Telnet / SSH]. Check `Allow SSH connection` and specify the port number (default 22). Next, check `Enable SFTP`. Once you click btn:[Apply], your admin user can log in to your NAS remotely.
+
+== Installation of ownCloud
+
+Go to the AppCenter and select QNAP Store.
+
+Search for ownCloud(_10.8.0_x86_64.qpkg) and click btn:[Install]. Confirm that you want to install this package. During installation you'll see a progress bar displaying the percentage.
+
+After a successful installation, ownCloud 10.8.0 will show up in the AppCenter under menu:QNAP Store[My Apps]. Click on btn:[Open] and proceed with the configuration.
+
+== Configuration
+
+=== First steps
+
+Log in as user `admin`. The default password is `admin` until you change it. In the upper right-hand corner, click on menu:Admin[Settings]. In the left hand navigation bar you find personal configuration options and admin-specific options. Under menu:Personal[General] you can change the password of you admin user.
+
+For more information on the Web interface, refer to the xref:user_manual:webinterface.adoc[Web UI] documentation.
+
+=== Configure your email.
+
+Email is an essential tool in ownCloud for notifying users, user registration, password resets, etc. Logged-in as admin, you can configure email in the menu:Admin[General] settings.
+
+For more information, refer to the section xref:configuration/server/email_configuration.adoc[Email Configuration].
+
+=== Cron
+
+On the same page (menu:Admin[General]) you find the Cron settings.
+
+Make sure Cron remains enabled. This daemon takes care of service jobs in 15-minute intervals.
+
+NOTE: In general, we recommend to keep the default settings in the admin section.
```

Comment:
For people less familiar with ownCloud this might be confusing.

I would propose something like that:
```
ownCloud performs regular tasks. In order to schedule them a mechanism called "cron" is used. It's already set up for you so you don't need to care about it. Please make sure to not change cron related settings except for a reason.
```

Reply:
Okay.